mission/config/artillery.hpp

```diff
@@ -7,7 +7,7 @@ class vn_artillery_settings
         availability[] = {0, 1, 1, 0, 0};
         unit_trait_required = 1;
 	      // Distance from the edge of a blacklisted marker that a artillery/aircraft cannot be called in.
-	      danger_distance = 150;
+	      danger_distance = "['danger_close_distance', 150] call BIS_fnc_getParamValue";
```

Comment:
i'm not sure how i feel about parameters being read via script code living into a config file.....

but i'm guessing this config is loaded by the CDLC directly, and there's no real other option to configure the arty module?

(I've never used the arty module, always played with it disabled)

Reply:
Not sure about the specifics of it, but I believe I followed how the other similar parameters are set in Mike Force. (Been quite some time since I made this change and don't remember it that well 😅